From 000543280335df4840eac1314b4938a8e174b638 Mon Sep 17 00:00:00 2001
From: Pillar <zuobaozhu@gmail.com>
Date: Thu, 24 Mar 2016 11:55:47 +0800
Subject: [PATCH 5/5] Bug Fix: TTY buffer overrun

Increase the TTY input buffer size to compensate for high latency
response from the BT stack.
---
 drivers/tty/tty_buffer.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/tty/tty_buffer.c b/drivers/tty/tty_buffer.c
index 2f78b77..a1ebe03 100644
--- a/drivers/tty/tty_buffer.c
+++ b/drivers/tty/tty_buffer.c
@@ -25,7 +25,7 @@
  * Byte threshold to limit memory consumption for flip buffers.
  * The actual memory limit is > 2x this amount.
  */
-#define TTYB_DEFAULT_MEM_LIMIT	65536
+#define TTYB_DEFAULT_MEM_LIMIT	(65536 * 16)
 
 /*
  * We default to dicing tty buffer allocations to this many characters
-- 
2.8.0.rc3

